Packs/Gmail/Integrations/Gmail/README.md

```diff
@@ -5044,3 +5004,121 @@ This command is only available to service account clients who have been delegate
 >|---|---|
 >| user2@domain.io | false |
 
+
+### gmail-forwarding-address-add
+***
+Creates a forwarding address. If ownership verification is required, a message will be sent to the recipient and the resource's verification status will be set to pending; otherwise, the resource will be created with verification status set to accepted. This method is only available to service account clients that have been delegated domain-wide authority. The special value "me" can be used to indicate the authenticated user.
+
+
+#### Base Command
+
+`gmail-forwarding-address-add`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
```

Comment:
Need to update the readme with the new changes and remove what is not necessary (like the disposition here)